sofa-ark-parent/core-impl/container/src/main/java/com/alipay/sofa/ark/container/pipeline/RegisterServiceStage.java

```diff
@@ -50,6 +52,8 @@ public void process(PipelineContext pipelineContext) throws ArkException {
     private void registryDefaultService() {
         registryService.publishService(BizDeployer.class, new DefaultBizDeployer(),
             new ContainerServiceProvider());
+        registryService.publishService(CommandProvider.class, new PluginCommandProvider(),
+            new ContainerServiceProvider());
```

Comment:
it's better use a uniqueId to publish a service, as maybe multi implementation of `CommandProvider` would be supported.